package.json

```diff
@@ -34,7 +34,7 @@
     "url": "git+https://github.com/getredash/redash.git"
   },
   "engines": {
-    "node": ">14.16.0 <17.0.0",
+    "node": ">14.16.0 <19.0.0",
```

Comment:
Just noticed this.  Is this change needed in this PR?

Reply:
nope, it was needed only for a local purposes, I have nodejs 18 on my system, reverted this change

Reply:
@justinclift it's now green

Reply:
Thanks for keeping an eye on it, I'll merge it now. :smile: